src/test/resources/alinatinca/suite.xml

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
Please don't delete this suite.xml as it was used for Tema 7. Add it in the same directory with the one for desktop and mobile. Thanks!